base/dft.jl

```diff
@@ -395,8 +395,17 @@ if Base.USE_GPL_LIBS
 
     A multidimensional FFT simply performs this operation along each transformed dimension of `A`.
 
-    Higher performance is usually possible with multi-threading. Use `FFTW.set_num_threads(np)`
-    to use `np` threads, if you have `np` processors.
+    Notes
+    =====
+        
```

Comment:
please run `make check-whitespace` locally, otherwise adding `[ci skip]` is dangerous